### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/astrocommunity#54] [Announcements]
Initial release of AstroCommunity! This is meant to indicate that the organization and structure of this repository has reached a good state and we can start having semantic versioning to track breaking changes and protect users from things going away or messing up their configuration.

This initial release has 55 plugins including 10 colorschemes and 6 language packs!

## What's Changed
* feat(media): add pets by @luxus in https://github.com/AstroNvim/astrocommunity/pull/6
* adding priority and disable lazy by @luxus in https://github.com/AstroNvim/astrocommunity/pull/1
* feat(media) add presence by @luxus in https://github.com/AstroNvim/astrocommunity/pull/3
* feat(color): adding tint by @luxus in https://github.com/AstroNvim/astrocommunity/pull/4
* feat(scrolling): Mini.animate by @luxus in https://github.com/AstroNvim/astrocommunity/pull/5
* feat(media): adding Wakatime by @luxus in https://github.com/AstroNvim/astrocommunity/pull/7
* feat(colorscheme): Everforest by @luxus in https://github.com/AstroNvim/astrocommunity/pull/8
* feat(scrolling): Adding Satellite.nvim by @luxus in https://github.com/AstroNvim/astrocommunity/pull/9
* feat(diagnostics): add trouble by @luxus in https://github.com/AstroNvim/astrocommunity/pull/10
* feat(colorscheme): Add Kanagawa by @luxus in https://github.com/AstroNvim/astrocommunity/pull/11
* feat(media): drop-nvim by @luxus in https://github.com/AstroNvim/astrocommunity/pull/12
* feat(pack): Go by @luxus in https://github.com/AstroNvim/astrocommunity/pull/13
* fix(completion): Renaming Copilot and adding copilot without cmp by @luxus in https://github.com/AstroNvim/astrocommunity/pull/14
* feat(pack) add rust by @luxus in https://github.com/AstroNvim/astrocommunity/pull/2
* feat(motion): add mini.bracketed by @luxus in https://github.com/AstroNvim/astrocommunity/pull/17
* Small Cleanup by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/18
* chore(ci): add conventional commit check by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/19
* Adding multiple colorschemes and remove lazyloading by @luxus in https://github.com/AstroNvim/astrocommunity/pull/23
* chore: various code cleanup by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/22
* chore: more various code cleanup by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/24
* More plugin additions by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/25
* chore(docs): improve readme colors by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/26
* feat(utility): add noice by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/27
* fix: update core modules to reflect new astronvim refactor by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/28
* fix(media): change the presence ID to astronvim by @luxus in https://github.com/AstroNvim/astrocommunity/pull/29
* Some more packs by @luxus in https://github.com/AstroNvim/astrocommunity/pull/30
* feat: Add Neogen, Dial, spectre and neodim by @luxus in https://github.com/AstroNvim/astrocommunity/pull/31
* feat(motion): Add Mini.Basics by @luxus in https://github.com/AstroNvim/astrocommunity/pull/33
* feat(motion): Add Mini.move by @luxus in https://github.com/AstroNvim/astrocommunity/pull/34
* feat(motion): Add Portal-nvim by @luxus in https://github.com/AstroNvim/astrocommunity/pull/35
* chore(docs): update the README and add CONTRIBUTING guidelines by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/37
* chore(docs): start adding READMEs to each plugin by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/38
* chore(docs): update readme plugin lists by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/39
* feat(colorscheme): Add Oxocarbon-nvim by @luxus in https://github.com/AstroNvim/astrocommunity/pull/40
* fix(pack): fix typescript language pack filetypes by @rafaelderolez in https://github.com/AstroNvim/astrocommunity/pull/42
* Added regexplainer by @owittek in https://github.com/AstroNvim/astrocommunity/pull/43
* 7 more plugins by @luxus in https://github.com/AstroNvim/astrocommunity/pull/45
* feat(python): add debugger by @utkarshgupta137 in https://github.com/AstroNvim/astrocommunity/pull/46
* fix(pack): using table.insert instead of list_extend on python dap by @luxus in https://github.com/AstroNvim/astrocommunity/pull/47
* fix: update to new user event based file lazy loading by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/48
* fix(motion): fix portal keybinding and run setup by @luxus in https://github.com/AstroNvim/astrocommunity/pull/49
* feat(pack): add Julia language pack by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/50
* fix(ccc): correct module path for highlighter by @A-Lamia in https://github.com/AstroNvim/astrocommunity/pull/52
* feat(pack): make language packs work without user intervention by @mehalter in https://github.com/AstroNvim/astrocommunity/pull/53

## New Contributors
* @mehalter made their first contribution in https://github.com/AstroNvim/astrocommunity/pull/18
* @rafaelderolez made their first contribution in https://github.com/AstroNvim/astrocommunity/pull/42
* @owittek made their first contribution in https://github.com/AstroNvim/astrocommunity/pull/43
* @utkarshgupta137 made their first contribution in https://github.com/AstroNvim/astrocommunity/pull/46
* @A-Lamia made their first contribution in https://github.com/AstroNvim/astrocommunity/pull/52

**Full Changelog**: https://github.com/AstroNvim/astrocommunity/commits/v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/astrocommunity/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by folke** [comment]
This is really cool. I was hoping that someone would create a repo like this using lazy's `import` feature :)

I assume most specs can be used outside of AstrNvim too?

Either way, great job! 🚀

**Reply by mehalter** [reply]
Thanks for the support @folke ! Yeah most of the specs are definitely cross compatible, but this isn't necessarily the case. The biggest gain we are trying to get from this is having the ability for plugin specs to not only configure the plugin they are setting up, but being able to reconfigure other plugins such as core AstroNvim features. Thanks for all your hard work with how the `lazy.nvim` configuration works because it has literally been a dream to work with and has really empowered AstroNvim !

The best example here I would say is the [copilot-lua-cmp pack](https://github.com/AstroNvim/astrocommunity/blob/main/lua/astrocommunity/completion/copilot-lua-cmp/copilot-lua-cmp.lua) pack where it not only sets up Copilot, but also reconfigured the default `cmp` configuration since copilot.lua expects to be able to use the `<Tab>` key by default :D Once one user figures out a good solution for the `cmp` configuration then they can immediately contribute it to the community !

**Reply by folke** [reply]
Nice! I added one for LzzyVim as well :) [copilot.lua](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/coding/copilot.lua)
